tests/plots/test_declarative.py

```diff
@@ -11,14 +11,16 @@
 import numpy as np
 import pandas as pd
 import pytest
+from shapely.geometry import (LineString, MultiLineString, MultiPoint, MultiPolygon, Point,
```

Comment:
These will need to move inside the test functions that need them so that our tests can continue working without Cartopy installed.